src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+    private static final int LOTTO_NUMBERS_COUNT = 6;
+    private final Set<LottoNumber> lottoNumbers;
```

Comment:
상수와 인스턴스 변수를 구별하기 쉽게  줄바꿈을 해주면 가독성을 조금 더 높일 수 있을 것 같아요. 🙂

```suggestion
    private static final int LOTTO_NUMBERS_COUNT = 6;
    
    private final Set<LottoNumber> lottoNumbers;
```